draft-ietf-ppm-dap.md

```diff
@@ -789,26 +791,21 @@ include any information specifying a particular batch (see {{query}}). The
 Leader selects a recent batch to aggregate. The Leader MUST select a batch that
 has not yet been associated with a collection job.
 
-In addition to the minimum batch size common to all query types, the
-configuration may include a parameter `max_batch_size` that determines maximum
-number of reports per batch.
-
-If the configuration does not include `max_batch_size`, then the Aggregators
-can output any batch size that is larger than or equal to `min_batch_size`.
-This is useful for applications that are not concerned with sample size, i.e.,
-the privacy guarantee is not affected by the sampling rate of the population,
-therefore a larger than expected batch size does not weaken the designed privacy
-guarantee.
+The Aggregators can output batches of any size that is larger than or equal to
+`min_batch_size`. This is useful for applications that are not concerned with
+sample size, i.e., the privacy guarantee is not affected by the sampling rate of
+the population, therefore a larger than expected batch size does not weaken the
+designed privacy guarantee.
 
-Implementation note: The goal for the Aggregators is to aggregate the same
-number of reports in each batch. The target batch size is deployment-specific,
+Implementation note: The target batch size, if any, is implementation-specific,
 and may be equal to or greater than the minimum batch size. Deciding how soon
-batches should be output is also deployment-specific. Exactly sizing batches may
-be challenging for Leader deployments in which multiple, independent nodes
+batches should be output is also implementation-specific. Exactly sizing batches
+may be challenging for Leader deployments in which multiple, independent nodes
 running the aggregate interaction (see {{aggregate-flow}}) need to be
-coordinated. The difference between the minimum batch size and maximum batch
-size is in part intended to allow room for error, and allow a range of target
-batch sizes.
+coordinated. Applications which include differential privacy may wish to control
+the size (or range of sizes) of generated batches in order to control the amount
+of noise added to reports by Clients (or added to aggregate shares by
+Aggregators).
```

Comment:
here too, @wangshan (see above comment)

Reply:
Similar to above, shall we just say whether to control the size of the batch is an implementation specific choice?